src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/events/CreateExceptionRecord.java

```diff
@@ -9,12 +9,19 @@
 @Component
 class CreateExceptionRecord extends AbstractEventPublisher {
 
+    static final String CASE_TYPE = "ExceptionRecord";
```

Comment:
can this be private ?

Reply:
It is used in DelegatePublisher and then in functional tests from there. Most of classes and it's elements are package private because it's not needed to be exposed. Everything orchestrated in orchestrator :D